app/mutations/EmailResendVerification.js

```diff
@@ -0,0 +1,34 @@
+import Relay from 'react-relay';
+
+class EmailResendVerification extends Relay.Mutation {
```

Comment:
We don't actually use this anymore, so perhaps it should be removed @keithpitt @ticky?